core/arch/x86/decode_table.c

```diff
@@ -4299,8 +4376,147 @@ const instr_info_t prefix_extensions[][8] = {
     {INVALID,      0x6638f618, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
     {OP_mulx,      0xf238f618, "mulx",    By, Gy, Ey, uDX, xx, mrm|vex, x, END_LIST},
   },
+  { /* prefix extension 144 */
+    {OP_seto,        0x0f9010, "seto",    Eb, xx, xx, xx, xx, mrm, fRO, END_LIST},
+    {INVALID,      0xf30f9010, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,      0x660f9010, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
```

Comment:
I don't think this is invalid: this is just "data16 seto" where the data16 is ignored.  I just tried it and it ran fine on my processor; plus multiple decoders accept it just fine:
```
$ echo " 0x66 0x0f 0x91 0x7d 0xfc" | /usr/bin/llvm-mc -arch x86 --disassemble
	.text
	setno	-4(%ebp)
$ echo " 0x66 0x0f 0x91 0x7d 0xfc" | /extsw/pkgs/disasm/capstone/build/capstone -x86 - 
+0x0000  660f917dfc   setno	byte ptr [ebp - 4]
(gdb) set {unsigned char[400]}0x04311000 = { 66, f, 90, 7d, fc, 90, 90, 90, 90, 90, 90 }
(gdb) x/3i 0x04311000
   0x4311000:	data16 seto -0x4(%rbp)
   0x4311005:	nop
```

I think this needs to instead split by vex encoding to distinguish seto from these new instructions, instead of splitting by prefix first.

Reply:
If that's the case I can go through VEX_EXT first. Done. The decoder now checks for PREFIX_EXT after VEXT_EXT, which is needed for the new instructions.